eccehomo/test_settings.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+TEST_MEDIA_ROOT = os.path.join(os.getcwd(), 'test_media')
```

Comment:
I think this is better on the OS temp folder, cleaning it at the end of the tests.